capa-controller-role/policies/crossplane-policy.json

```diff
@@ -14,7 +14,18 @@
                 "ec2:RevokeSecurityGroupIngress",
                 "cloudwatch:*",
                 "sqs:*",
-                "events:*"
+                "events:*",
+                "iam:GetRole",
+                "iam:CreateRole",
+                "iam:DeleteRole",
+                "iam:TagRole",
+                "iam:GetRolePolicy",
+                "iam:PutRolePolicy",
+                "iam:ListRolePolicies",
+                "iam:DeleteRolePolicy",
+                "iam:AttachRolePolicy",
+                "iam:DetachRolePolicy",
+                "iam:ListInstanceProfilesForRole"
```

Comment:
This is definitely not a comprehensive list of permissions required by crossplane but it's a start. It currently works because coincidentally [the `iam-controller-policy`](https://github.com/giantswarm/giantswarm-aws-account-prerequisites/blob/main/capa-controller-role/policies/iam-controller-policy.json) adds the required `iam` permissions. It would be good to be specific and add all the permissions required by crossplane, avoiding using wildcards like `iam:*`.